hieradata/site/cp/role/nfsserver.yaml

```diff
@@ -77,6 +77,13 @@ nfs::nfs_exports_global:
       139.229.165.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.170.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.191.0/25(rw,nohide,insecure,no_subtree_check,async,root_squash)
+  /data/rsphome:
```

Comment:
Is this going to be migrated to the new storage cluster or should it be hosted on yagan?  I don't think we have a clear plan for what we are going to do with nfs exports.